### PR TITLE
feat: support filters on web

### PR DESCRIPTION
**Pull request by jakex7**
# Summary

* Export `Filter` and `FeColorMatrix` components on `web`
* Change filter IDs in example to be unique*
* Generate filter ID when using `FilterImage`*
* Hide `FilterImage` example on web, since it's crashing the whole site (see #2334)

\* ID on web has to be unique, otherwise it'll use the first element with that ID, even if they are in the separate SVG elements

## Test Plan

run `web-example` app

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| Web     |    ✅     |

### CODE REVIEW

**Review by WoLewicki — COMMENTED** [review]
Should we wait for https://github.com/software-mansion/react-native-svg/pull/2334 with merging this one?

**Comment by jakex7** [comment]
> Should we wait for #2334 with merging this one?

@WoLewicki 
We don't have to. It's temporary disabling `FilterImage` example on web so everything else should work.

File: src/filter-image/FilterImage.tsx [hunk 1]
```diff
@@ -19,13 +26,32 @@ const getFilters = (filters: FilterImageProps['filters']) => {
...
+        ';',
+      ''
+    )
+  );
+};
+
 export const FilterImage = (props: FilterImageProps) => {
   const { source, style, ...imageProps } = props;
+  const hasFilter = props.filters.length > 0;
+  const filterId = useMemo(() => getFilterName(props.filters), [props.filters]);
```
Comment (WoLewicki) [inline src/filter-image/FilterImage.tsx:46]: What if all the filters are the same on both images? Wouldn't it be better to just use some hash and add it?

Comment (jakex7) [reply src/filter-image/FilterImage.tsx:46]: Sure, any suggestions on how to get a relatively unique hash? I'm afraid that `Math.random` won't be sufficient.

Comment (jakex7) [reply src/filter-image/FilterImage.tsx]: Done in [26cbf53](https://github.com/software-mansion/react-native-svg/pull/2346/commits/26cbf53ee2f9566c98eb2d8afc729eb4d9a16290)

File: src/filter-image/FilterImage.tsx [hunk 2]
```diff
@@ -20,12 +29,17 @@ const getFilters = (filters: FilterImageProps['filters']) => {
 };
 
 export const FilterImage = (props: FilterImageProps) => {
-  const { source, style, ...imageProps } = props;
+  const { filters, source, style, ...imageProps } = props;
+  const hasFilter = filters.length > 0;
+  const filterId = useMemo(() => `RNSVG-${getRandomNumber()}`, [filters]);
```
Comment (WoLewicki) [inline src/filter-image/FilterImage.tsx:34]: Do we need to change the `filterId` when the `filters` change?

**Review by WoLewicki — APPROVED** [review]
Added one comment. Please answer it and then I think we are good to go 🚀

File: src/filter-image/FilterImage.tsx (hunk 2 above)
Comment (jakex7) [reply src/filter-image/FilterImage.tsx]: After some tests, we don't. Removed in https://github.com/software-mansion/react-native-svg/pull/2346/commits/af12b62af91bc39b87f1b82630eb08233793f657

**Pull request merged**